Limits of sequences in metric spaces (#1378)
UniMath/agda-unimath: src/metric-spaces/sequences-metric-spaces.lagda.md

Reviewer: `metric-space-of-sequences-metric-spaces`

# Sequences in metric spaces

```agda
module metric-spaces.sequences-metric-spaces where
```

<details><summary>Imports</summary>

```agda
open import elementary-number-theory.natural-numbers

open import foundation.universe-levels

open import metric-spaces.dependent-products-metric-spaces
open import metric-spaces.metric-spaces
```

</details>

## Idea

> The type of [sequences](foundation.sequences.md) in the underlying type of a

Reviewer: This file seems to be first and foremost about the concept of sequences in a metric space, so you should define that concept first in this idea section. You're welcome to then also state that it defines a metric space

> The type of [sequences](foundation.sequences.md) in the underlying type of a
> [metric space](metric-spaces.metric-spaces.md) inherits the

Reviewer: ```suggestion
The type of [sequences](foundation.sequences.md) in a
[metric space](metric-spaces.metric-spaces.md) inherits the
```

> [product metric structure](metric-spaces.dependent-products-metric-spaces.md);
> it is the
> {{#concept "metric space of sequences" Disambiguation="in a metric space" Agda=sequence-Metric-Space}}
> in a metric space.

Reviewer: ```suggestion
[product metric structure](metric-spaces.dependent-products-metric-spaces.md).
This defines the
{{#concept "metric space of sequences" Disambiguation="in a metric space" Agda=sequence-Metric-Space}}
in a metric space.
```
